controllers/controllers.js

```diff
@@ -71,5 +71,21 @@ exports.getComments = (req, res, next) =>{
   .catch((err) =>{
     next(err)
   });
+};
 
+//========================================================================
+exports.postComment = (req, res, next) => {
+  const articleId = req.params.article_id;
+  const {username, body} = req.body;
+  if (!username || !body){
+    err = { ststus: 400, msg: "Bad Request"}
```

Comment:
Typo here: `ststus` 😄

Reply:
Hi Karla
Could you please explain more? This check for username and body have to be string not number

Reply:
&& instead of ||
and instead of or